docs/plugins.md

```diff
@@ -0,0 +1,351 @@
+      request.responseHeaders.some(header => header.name.toLowerCase() === 'x-debug-data')
+    );
+
+    return {
+      score: badRequests.length === 0 ? 1 : 0,
+    };
+  }
+}
+
+module.exports = HeaderPoliceAudit;
+```
+
+## Best Practices
+
+### Naming
+
+> There are only two hard things in Computer Science: cache invalidation and naming things.
+> Phil Karlton
+
+There are several display strings you will need to write in the course of plugin development. To ensure your plugin users have a consistent experience with the rest of the Lighthouse report, follow these guidelines.
+
+#### Category Titles
+
+Write category titles that are short (fewer than 20 characters), ideally a single word or acronym. Avoid unnecessary prefixes like "Lighthouse" or "Plugin" which will already be clear from the context of the report.
+
+#### Category Descriptions
+
+Write category descriptions that provide context for your plugin's audits and link to where users can learn more or ask questions about their advice.
+
+#### Audit Titles
+
+Write audit titles in the _present_ tense that _describe_ what the page is successfully or unsuccessfully doing.
+
+**DO**
+
+> Document has a `<title>` element
+
+> Document does not have a `<title>` element
+
+> Uses HTTPS
+
+> Does not use HTTPS
+
+> Tap targets are sized appropriately
+
+> Tap targets are not sized appropriately
+
+**DON'T**
+
+> Good job on `alt` attributes
+
+> Fix your headers
+
+#### Audit Descriptions
+
+Write audit descriptions that provide brief context for why the audit is important and link to more detailed guides on how to follow its advice. Markdown links are supported, so use them!
+
+**DO**
+
+> Interactive elements like buttons and links should be large enough (48x48px), and have enough space around them, to be easy enough to tap without overlapping onto other elements. [Learn more](https://developers.google.com/web/fundamentals/accessibility/accessible-styles#multi-device_responsive_design).
+
+> All sites should be protected with HTTPS, even ones that don\'t handle sensitive data. HTTPS prevents intruders from tampering with or passively listening in on the communications between your app and your users, and is a prerequisite for HTTP/2 and many new web platform APIs. [Learn more](https://developers.google.com/web/tools/lighthouse/audits/https).
+
+**DON'T**
+
+> Images need alt attributes.
+
+> 4.8.4.4 Requirements for providing text to act as an alternative for images
+> Except where otherwise specified, the alt attribute.... 10,000 words later... and that is everything you need to know about the `alt` attribute!
+
+### Scoring
+
+1. Weight each audit by its importance.
+1. Differentiate scores within an audit by returning a number _between_ `0` and `1`. Scores greater than `0.9` will be hidden in "Passed Audits" section by default.
+1. Don't inflate scores unnecessarily by marking audits as not applicable. When an audit's advice doesn't apply, simply `return {notApplicable: true}`.
```

Comment:
```suggestion
1. Avoid inflating scores unnecessarily by marking audits as not applicable. When an audit's advice doesn't apply, simply `return {notApplicable: true}`.
```
otherwise it could be read as don't mark things as not applicable because it inflates the scores unnecessarily :)